src/machine/board_rp2040.go

```diff
@@ -0,0 +1,41 @@
+// +build rp,rp2040 arduino_nano_rp2040_connect pico
```

Comment:
This applies to all rp2040?  If so should build tag just be:
`// +build rp2040`

Reply:
I have no opinion, was following this example
https://github.com/tinygo-org/tinygo/blob/release/src/machine/board_atsamd21.go#L1
No prob to change, what shall it be then?

Reply:
I'd err on the side of simplicity, unless some other folks want to jump in with other thoughts.

Reply:
OK, let's wait a day, if no-one objects, I go and simplify to just "rp2040"

Reply:
I agree that a single tag is best when possible.